spec.bs

```diff
@@ -2892,9 +2933,9 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
 
       1. Run steps in |config|'s [=fenced frame config/on navigate callback=].
   
-    1. If |navigable| is a [=fenced navigable container/fenced navigable=] and
-       <var ignore>sourceDocument</var>'s [=node navigable=] is in |navigable|'s
-       [=navigable/active document=]'s [=Document/ancestor navigables=]:
+    1. If |navigable| is a [=fenced navigable container/fenced navigable=] and <var
+       ignore>sourceDocument</var>'s [=node navigable=] is in |navigable|'s [=navigable/active
+       document=]'s [=Document/ancestor navigables=] with [=an-unfenced|unfenced=] set to true:
```

Comment:
I think this part (i.e., this usage of "ancestor navigables") is fine though, since it doesn't have to do with gating the access of the fenced frame URN config mapping. Do you agree?

Reply:
I think this is fine. This is also needed because we only want to create a fenced frame config if the navigation is initiated by the fenced frame's embedder. If a fenced frame's document navigates itself, that should just re-use the existing config.